packages/code-studio/src/settings/FormattingSectionContent.tsx

```diff
@@ -114,7 +100,7 @@ export class FormattingSectionContent extends PureComponent<
       this.commitChanges.bind(this),
```

Comment:
Above - you no longer need the `defaults` set in the `defaultProps`, since it's passed in via redux